src/main/ArrayManager.java

```diff
@@ -178,14 +178,11 @@ public void shuffleArray(int[] array, int currentLen, ArrayVisualizer ArrayVisua
 
         if(ArrayVisualizer.isActive()) {
             double sleepRatio = ArrayVisualizer.getCurrentLength()/1024d;
+            sleepRatio *= shuffle.sleepRatio;
```

Comment:
this system forces the user to manually adjust for the number of shuffles, imo this plus a checkbox for total time / time per shuffle is better

Reply:
Good idea!